### PR TITLE
Add ACME tests with postgresql back-end

### PR DESCRIPTION
**Pull request by fmarco76**

### CODE REVIEW

**Review by edewata — APPROVED** [review]
LGTM, thanks! It might be nice to add some steps to verify that the data got stored properly in PostgreSQL, but that can be added separately later.

File: .github/workflows/acme-tests.yml [hunk 1]
```diff
@@ -51,3 +51,10 @@ jobs:
     uses: ./.github/workflows/acme-container-test.yml
     with:
       db-image: ${{ needs.init.outputs.db-image }}
+
+  acme-postgrsql-test:
+    name: ACME with postgrsql back-end
```
Comment (edewata) [inline .github/workflows/acme-tests.yml:56]: Ah, typo?

**Review by ckelleyRH — APPROVED** [review]
LGTM!

**Comment by fmarco76** [comment]
> verify that the data got stored properly in PostgreSQL

Added the same tests in `ACME with certbot` work-flow to verify the information stored in the DB during the registration and the enrolment

**Comment by fmarco76** [comment]
> LGTM! There is a failure in the test though, so once that is resolved it should be OK.

The workflow is not running correctly but there is a failure in the workflow CA with a problem on getting docker image. Since the error cannot be related to this PR I'll merge this and check the error in the other workflow if it is present in other PRs.

**Comment by fmarco76** [comment]
@edewata @ckelleyRH Thanks!

**Pull request merged**